datasource/src/test/scala/quasar/physical/mongo/MongoDataSourceModuleSpec.scala

```diff
@@ -44,14 +44,4 @@ class MongoDataSourceModuleSpec extends EffectfulQSpec[IO] {
     RateLimiter[IO, UUID](1.0, IO.delay(UUID.randomUUID()), NoopRateLimitUpdater[IO, UUID]).flatMap(rl =>
       MongoDataSourceModule.lightweightDatasource[IO, UUID](config, rl, ByteStore.void[IO], _ => IO(None)).use(r => IO(r must beRight)))
   }
-
-  "Using unreachable config produces Left invalid configuration" >>* {
```

Comment:
Maybe still useful to keep around but with altered expectation?

Reply:
:+1: